lib/core/typeinfo.nim

```diff
@@ -282,7 +282,7 @@ proc `[]=`*(x: Any, i: int, y: Any) =
   else: assert false
 
 proc len*(x: Any): int =
-  ## len for an any `x` that represents an array or a sequence.
+  ## ``len`` for an any `x` that represents an array or a sequence.
```

Comment:
nit: as mentioned in contributing.rst, prefer single backticks to double backticks since it's simpler and it's supported by nim doc/rst2html, and it's what ppl are most familiar with (markdown)